docs/source/nlp/quantization.rst

```diff
@@ -133,15 +132,60 @@ Known issues
 * The supported and tested model family is Llama2. Quantizing other model types is experimental and may not be fully supported.
 
 
-Please refer to the following papers for more details on quantization techniques.
+Quantization-Aware Training (QAT)
+---------------------------------
 
-References
-----------
+QAT is the technique of fine-tuning a quantized model to recover model quality degradation due to quantization.
+While QAT requires much more compute resources than PTQ, it is highly effective in recovering model quality.
+To perform QAT on a calibrated model from PTQ, you need to further fine-tune the model on a downstream task using a small dataset before exporting to TensorRT-LLM.
+You can reuse your training pipeline for QAT.
+Since QAT is done after PTQ, the supported model families are the same as for PTQ.
```

Comment:
Can we redirect the users to https://github.com/NVIDIA/TensorRT-Model-Optimizer/tree/main/llm_qat#qat-example-workflow

or copy-paste the example workflow to here?

```suggestion
QAT is the technique of fine-tuning a quantized model to recover model quality degradation due to quantization. During QAT, the scaling factors inside quantizers are frozen and the model weights are fine-tuned.

Please see [QAT Example Workflow](https://github.com/NVIDIA/TensorRT-Model-Optimizer/tree/main/llm_qat#qat-example-workflow) for more details on QAT workflow.
```

Reply:
The way PTQ example was previously authored by @janekl is using a Quantizer class which abstracts usage of `modelopt` from the user so I would prefer to not expose nemo users to this doc as it may overwhelm them. Let me add the note on scaling factors and weight update as you suggested. Users can take a look at the github repository root already linked above and navigate to QAT if they are interested to have a deeper idea.

Wdyt?

Reply:
Lets update the documentation here itself.